ocaml/xapi-consts/api_messages.ml

```diff
@@ -132,6 +132,7 @@ let host_cpu_features_down = addMessage "HOST_CPU_FEATURES_DOWN" 3L
 let host_cpu_features_up = addMessage "HOST_CPU_FEATURES_UP" 5L
 let pool_cpu_features_down = addMessage "POOL_CPU_FEATURES_DOWN" 5L
 let pool_cpu_features_up = addMessage "POOL_CPU_FEATURES_UP" 5L
+let host_low_memory = addMessage "HOST_LOW_MEMORY" 2L
```

Comment:
This isn't quite precise enough. The condition is that Xen itself is low on memory. There may still be free memory for VMs. So perhaps `XEN_LOW_MEMORY`.

Reply:
Isn't it true that free memory of xen equals to free memory of the host?
When I test the implementation, I continuously create VMs on the host to consume the memory and finally trigger the VIRQ_ENOMEM event of xen.

Reply:
I'm sure entirely sure how Xen's own memory management works. Perhaps @andyhhp could make a suggestion?

Reply:
Thanks, so @krizex is right.